pkg/driver/driver_aws.go

```diff
@@ -289,14 +289,20 @@ func (d *AWSDriver) Delete(machineID string) error {
 		if err != nil {
 			if aerr, ok := err.(awserr.Error); ok {
 				switch aerr.Code() {
+				case "InvalidInstanceAttributeValue":
+					// Case when disk is not yet attached to the VM
+					klog.Warning(aerr.Error())
+					break
```

Comment:
Now we are also not adding the API request failure count to the metrics. Just want to make sure that is a conscious decision. Otherwise lgtm.

Reply:
Valid point. Even i wasn't sure if this should be considered as failed or not.